sphinx_rtd_theme/static/js/app.js

```diff
@@ -36,36 +35,48 @@ $(document).ready(function () {
     });
   }
 
+  /**
+   * Chatlio.com integration
+   */
   function initChatlio() {
-    // Chatlio.com integration
     document.addEventListener('chatlio.ready', function () {
-      window._chatlio.configure({
+      if (lscache.get('me')) {
+        var me = lscache.get('me');
+        var userId;
+        if (me.name && me.name.indexOf(' ') !== -1) {
+          userId = me.name.substring(0, me.name.indexOf(' ') + 2).replace(/ /g, '').toLowerCase();
```

Comment:
Der er slet ingen grund til at ha denne kode længere, for det er jo hvad chatlio gør når vi giver dem et name felt, jvf. https://chatlio.com/docs/api-v1/#api-identify